canvas_modules/harness/assets/styles/common.scss

```diff
@@ -13,9 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-@use "@carbon/react" as * with (
-	$font-path: "/fonts"
+@use "@carbon/react" with (
+	$font-path: "/fonts",
+	$use-per-family-plex: true
```

Comment:
@tomlyn This is importing @ibm/plex fonts -
<img width="1715" alt="Screenshot 2025-06-23 at 11 57 51 AM" src="https://github.com/user-attachments/assets/3c83fbc7-6bb5-45a2-a31e-4dfe949365fd" />

Reply:
@nmgokhale I am seeing this now
<img width="679" alt="image" src="https://github.com/user-attachments/assets/3f01e8b5-89fc-481f-9baa-e7bb12cb7e97" />

Reply:
@nmgokhale 
I am seeing that error in the console after rebuilding and when viewing the `WYSIWYG Comments` sample app.

In addition, regarding fonts with this work we'll also need to update this in the docs
https://elyra-ai.github.io/canvas/02-set-up/#loading-fonts

Reply:
Created a separate issue for @ibm/plex fonts related changes. Right now `@import` is used in 2 files which will be addressed in the new issue -

![Screenshot 2025-07-03 at 12 13 00 PM](https://github.com/user-attachments/assets/d80918cc-0563-46c7-a5cf-420d50ef1c40)